test/generator/test_base_patches_factory.py

```diff
@@ -289,7 +293,9 @@ def test_create_base_patches(mocker):
         game, 0, layout_configuration,
     )
 
-    mock_add_elevator_connections_to_patches.assert_called_once_with(factory, layout_configuration, rng, patches[0])
+
+    mock_dock_connections_assignment.assert_called_once_with(factory, layout_configuration, game, rng)
+    # patches[0].assign_node_configuration.assert_called_once_with(mock_gate_assignment_for_configuration.return_value)
```

Comment:
why the commented line?

Reply:
Oops. Looks like I was distracted. This line should be activated and also changed for the call.
Addressed it